src/ape_pm/_cli.py

```diff
@@ -148,6 +149,67 @@ def install(cli_ctx, package, name, version, ref, force):
         cli_ctx.logger.success("All project packages installed.")
 
 
+@cli.command()
+@ape_cli_context()
+@click.argument("package", nargs=1, required=True)
+@click.argument("versions", nargs=-1, required=False)
+@click.option(
+    "-y", "--yes", is_flag=True, help="Automatically confirm the removal of the package(s)"
+)
+def remove(cli_ctx, package, versions, yes):
+    """
+    Remove a package.
```

Comment:
this is just a nit for consistency, because i noticed none of the other commands have a period at the end of their short-help

```suggestion
    Remove a package
```

Reply:
this comment got marked resolved but the period remains in the diff.
I unmarked it as resolved.